docs/static/releasenotes.asciidoc

```diff
@@ -94,6 +95,18 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha2,Logstash 8.0.0-alpha2>>
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
+[[logstash-8-19-3]]
+=== Logstash 8.19.3 Release Notes
+
+[[notable-8-19-3]]
+==== Notable issues fixed
```

Comment:
PR-17933 seems not an issue fix.
I would rather place this under "New features and enhancements" section.

Reply:
Udpated ✔️